Makefile

```diff
@@ -2,12 +2,8 @@ TESTS = test/*.js
 
 
 test:
-	@# if any of the files contain 'debugger' statements, start with --debug-brk
-	@if find -name 'node_modules' -prune -o -type f -name '*.js' | xargs grep -q '^\s*debugger'; then \
-		./node_modules/.bin/mocha --debug-brk $(TESTS); \
-		else \
-		./node_modules/.bin/mocha $(TESTS); \
-		fi
+	./node_modules/typescript/bin/tsc
+	./node_modules/mocha/bin/mocha
```

Comment:
Why did you remove the code to automatically add `--debug-brk`? It is actually an useful way to debug using mocha.

Also, the `tsc` command should probably go in a separate target:

```diff
diff --git a/Makefile b/Makefile
index 964cbaf..ea298e0 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,9 @@
 TESTS = test/*.js

+tsc:
+       @./node_modules/.bin/tsc

-test:
+test: tsc
        @# if any of the files contain 'debugger' statements, start with --debug-brk
        @if find -name 'node_modules' -prune -o -type f -name '*.js' | xargs grep -q '^\s*debugger'; then \
                ./node_modules/.bin/mocha --debug-brk $(TESTS); \
@@ -14,4 +16,4 @@ watch:
        @./node_modules/.bin/nodemon -e js --exec 'make test'


-.PHONY: test watch
+.PHONY: tsc test watch
```